### PR TITLE
Fixes #10090 - Minimised dashboard widged names are translated

### PR DESCRIPTION
**Pull request by dLobatog**

### CODE REVIEW

File: app/helpers/dashboard_helper.rb [hunk 1]
```diff
@@ -18,7 +18,8 @@ def render_widget(widget)
   end
 
   def widget_data(widget)
-    {:data=>{:id => widget.id, :name => widget.name, :row => widget.row, :col => widget.col, :sizex=>widget.sizex, :sizey => widget.sizey, :hide=>widget.hide}}
+    { :data => { :id    => widget.id,    :name  => _("#{widget.name}"), :row  => widget.row, :col => widget.col,
```
Comment (domcleal) [inline app/helpers/dashboard_helper.rb]: No need for string interpolation here, just `_(widget.name)`

**Comment by domcleal** [comment]
Do you know why we have different widget names (in app/services/dashboard/loader.rb) compared to the titles shown on the page?

**Comment by dLobatog** [comment]
@domcleal Not really, sorry. I'm guessing it's so that they read better in that dropdown? @tbrisker  any idea why?

**Comment by tbrisker** [comment]
@eLobato I just left the strings that were there before (so the UI change would be minimal), I'd guess you'd have to ask @abenari who wrote the original implementation.

**Comment by domcleal** [comment]
Merged as ae4d968, thanks @eLobato.

**Pull request closed without merging**